### ISSUE TITLE
color contrast check should be at least AA

### ISSUE THREAD

**Issue by yousseflaaroui3010** [yousseflaaroui3010/habitBuilder#19]
check in both dark/light mode all connected colors for color contrast ratio 

make sure that nothing is less than AA either text vs background, or logo vs background or button vs background....
